src/components/Editor/Tabs.css

```diff
@@ -49,26 +52,37 @@
   max-width: 100%;
   overflow: hidden;
   padding: 5px;
-  margin-inline-start: 3px;
-  margin-top: 3px;
   cursor: default;
-  height: 27px;
+  height: 28px;
+  font-size: 12px;
 }
 
 .source-tab:first-child {
   margin-inline-start: 0;
 }
 
-.source-tab:hover {
-  background-color: var(--theme-toolbar-background-alt);
-  border-color: var(--theme-splitter-color);
+.source-tab:not(.active):hover {
+  background-color: var(--theme-toolbar-hover);
 }
 
 .source-tab.active {
-  color: var(--theme-body-color);
-  background-color: var(--theme-body-background);
-  border-color: var(--theme-splitter-color);
+  color: rgb(0, 96, 223);
   border-bottom-color: transparent;
+  border-left: 1px solid var(--theme-splitter-color);
+  border-right: 1px solid var(--theme-splitter-color);
+
+  /* added this part to connect active file tab to editor */
+  background-color: var(--theme-body-background);
+  height: 30px;
+}
+
+/* added this part to balance out height of the not active file tabs */
+.source-tab:not(.active) {
+  bottom: 2px;
```

Comment:
Shouldn't the default state and active state have the same height? That's how the `devtools-tab` component is designed and implemented.